docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_gitops_basic/_index.md

```diff
@@ -193,7 +193,7 @@ With Namespace-level GitOps config, the goal is to have Kubernetes resources dep
   
     ![kubectl get pods -n hello-arc -w](./13.png)
 
-  - **Your fork** of the "Azure Arc Jumpstart" repository. Open the [*hello_arc.yaml*](https://github.com/zaidmohd/azure_arc/tree/main/artifacts/hello-arc/yaml/hello_arc.yaml) file.
+  - In **Your fork** of the "Azure Arc Jumpstart" repository, open the *hello_arc.yaml* file (/azure_arc/tree/main/artifacts/hello-arc/yaml/hello_arc.yaml).
```

Comment:
```suggestion
  - In **Your fork** of the "Azure Arc Jumpstart" repository, open the *hello_arc.yaml* file (/azure_arc/artifacts/hello-arc/yaml/hello_arc.yaml).
```

Reply:
Instead of replacing the original screenshot showing the values, keep it and then include the example values directly below with a note explaining that its an example.